### ISSUE TITLE
Galaxy role freeipa.ipaserver points to ipaclient, not ipaserver

### ISSUE THREAD

**Issue by jaredhocutt** [freeipa/ansible-freeipa#82]
When installing the freeipa.ipaserver role from Galaxy, it appears to be pointing to the ipclient role and not the ipaserver role.

**Comment by t-woerner** [comment]
How have you installed the freeipa.ipaserver role from Galaxy?

**Comment by jaredhocutt** [comment]
@t-woerner I am using a `requirements.yml` file and `ansible-galaxy` to install.

**requirements.yml**

```yaml
- src: freeipa.ipaserver
  version: v0.1.1
```

**Install**

`ansible-galaxy install -r requirements.yml`

**Comment by t-woerner** [comment]
I am able to reproduce it but do not understand why it happens.

**Comment by jaredhocutt** [comment]
I forked the repo and tried myself to see if I could figure out why it was happening, but also could not get it working as expected. The only way I was able to work around it was to split each role into it's own repo.

If I had to guess, this seems to be a bug in Galaxy since it does detect the multiple roles from the single repo but doesn't serve the correct one.

I think this issue can be closed for your repo since it doesn't appear to be a bug on your end. I'll file an issue with Galaxy.
